.github/workflows/main.yml

```
@@ -17,8 +17,8 @@ jobs:
         os:
           - ubuntu-18.04
           - ubuntu-20.04
-          - windows-2016
           - windows-2019
+          - windows-2022
           - macos-10.15
```

```suggestion
          - macos-10.15
          - macos-11
```
Might as well, maybe separately - reference https://github.com/actions/virtual-environments#available-environments

---

Yeah, agreed 👍 I thnk the same can go for Windows as well? :)